src/sass/_common/external/grid.scss

```diff
@@ -61,6 +61,10 @@
     padding-bottom: 30px;
 }
 
+.gr-bottom-30 {
+    padding-bottom: 30px;
```

Comment:
can you try using `gr-parent gr-padding-30` in conjunction? I think it should remove the padding top but keep the padding bottom 30 as you want